x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/type/DataType.java

```diff
@@ -101,6 +101,7 @@ public enum DataType {
     TEXT(builder().esType("text").unknownSize()),
     DATETIME(builder().esType("date").typeName("DATETIME").estimatedSize(Long.BYTES).docValues()),
     // IP addresses, both IPv4 and IPv6, are encoded using 16 bytes.
```

Comment:
Really minor: the comment belongs to the line below DATE_NANOS I think.

Reply:
oh, nice catch, thanks.